src/projects/projects.controller.ts

```diff
@@ -52,28 +52,55 @@ export class ProjectsController {
     */
   @Post()
   @UsePipes(new ValidationPipe())
-  async create(
-    @Body() request: CreateProjectDto,
-    @GetUser() user: RequestUser,
-  ): Promise<SanitizedProject> {
-    const createdProject = await this.projectsService.create(
-      user.id,
-      request.name,
-    );
-    return createdProject;
+  async create(@Body() request: CreateProjectDto, @GetUser() user: RequestUser): Promise<SanitizedProject> {
+    return this.projectsService.create(user.id, request.name);
   }
 
   // DELETE /projects/:uuid
   // This action deletes a #${id} project
   @Delete(':uuid')
   @UseGuards(JwtAuthGuard)
-  async delete(
-    @Param('uuid') uuid: string,
-    @GetUser() user: RequestUser,
-  ): Promise<SanitizedProject> {
-    // Delete the project from the database
-    const project = await this.projectsService.delete(uuid, user.id);
+  async delete(@Param('uuid') uuid: string, @GetUser() user: RequestUser): Promise<SanitizedProject> {
+    return this.projectsService.delete(uuid, user.id);
+  }
```

Comment:
When we delete a project, the container and image are removed properly, but the git repository is not removed so we get an error(?)